lms/data_tasks/report/create_from_scratch/99_grant_permissions/01_grant_schema.jinja2.sql

```diff
@@ -18,12 +18,14 @@
 
     GRANT USAGE ON SCHEMA report TO "{{fdw_user}}";
 
-    GRANT SELECT ON report.organization_activity TO "{{fdw_user}}";
-    GRANT SELECT ON report.organization_roles TO "{{fdw_user}}";
+    GRANT SELECT ON report.events TO "{{fdw_user}}";
     GRANT SELECT ON report.groups TO "{{fdw_user}}";
     GRANT SELECT ON report.group_map TO "{{fdw_user}}";
     GRANT SELECT ON report.group_bubbled_activity TO "{{fdw_user}}";
     GRANT SELECT ON report.group_bubbled_counts TO "{{fdw_user}}";
     GRANT SELECT ON report.group_roles TO "{{fdw_user}}";
+    GRANT SELECT ON report.organization_activity TO "{{fdw_user}}";
+    GRANT SELECT ON report.organization_roles TO "{{fdw_user}}";
+    GRANT SELECT ON report.users TO "{{fdw_user}}";
```

Comment:
I've also sorted this alphabetically. The change is the addition of `events` and `users`.